Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/AppCmdOnTargetMachines.ps1

```diff
@@ -100,7 +100,7 @@ function Does-BindingExists
     Write-Verbose "Checking binding exists for website (`"$siteName`"). Running command : $command"
 
     $sites = Run-Command -command $command -failOnErr $false
-    $binding = [string]::Format("{0}/{1}:{2}:{3}", $protocol, $ipAddress, $port, $hostname)
+    $binding = [string]::Format("{0}/{1}:{2}:{3},", $protocol, $ipAddress, $port, $hostname)
```

Comment:
Thanks for your contribution.. Few items that you need to take care

1. Increase patch version in task.json
2. Increase patch extension version in vss-extension.json
3. Update change Unreleased section